<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="xianshi">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>351</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="input">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>351</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="port_select">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="bolt_select">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>100</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="update_port">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_port">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
